docs/pages/web/text.js

```diff
@@ -185,6 +186,15 @@ export default function TextPage({ generatedDocGen }: {| generatedDocGen: DocGen
           title="Text-wrapping and hyphenation"
           description="Hyphenation on iOS is turned off by default to avoid incorrect word breaks when strings of text wrap to the next line. This is especially helpful for international languages where an incorrect word break can greatly change the meaning of a word or sentence."
         />
+
+        <MainSection.Subsection
+          title="Word Breaks and CJK languages"
```

Comment:
This isn't necessarily a variant right? It's more like a _test_ example.

Would "Word breaks" be enough if we add it under localization and we specify in the description what CJK languages stands for?

Reply:
I kinda fill this should go on the Quality Checklist under "Internationalization", kinda like RTL.

Reply:
Or even here: Foundations > Typography

https://gestalt.pinterest.systems/foundations/typography/guidelines#Best-practices

Reply:
I think the examples would be great in best practices here.

If Text handles it all internally, it shuoldn't be a variant.

Reply:
Agree! I'm not even sure what the best practice here is though. It's just a consideration maybe?

Reply:
Since this `keep-all` originates in the text component, I think it's good to call out here